GitUp/Application/Document.m

```diff
@@ -880,11 +886,11 @@ - (void)_showHelpWithIdentifier:(NSString*)identifier {
     XLOG_DEBUG_UNREACHABLE();
   }
   if (showHelp) {
-    NSRect contentBounds = [_mainWindow.contentView bounds];
```

Comment:
I tried 6 and 7 and it seems to work.

`window.contentView.bounds` didn't work because `contentView` was `id` until the 10.11 SDK (when it becomes `__kindof NSView*`).